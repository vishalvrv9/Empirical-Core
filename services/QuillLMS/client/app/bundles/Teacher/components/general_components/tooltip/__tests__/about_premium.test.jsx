import React from 'react';
import { shallow } from 'enzyme';

import AboutPremium from '../about_premium';

describe('AboutPremium component', () => {

  it('should render', () => {
    expect(shallow(<AboutPremium />)).toMatchSnapshot();
  });

});
