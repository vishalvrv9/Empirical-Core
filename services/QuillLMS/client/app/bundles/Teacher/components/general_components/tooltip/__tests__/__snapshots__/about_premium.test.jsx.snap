// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutPremium component should render 1`] = `
<div
  className="advert"
>
  <div
    className="advert-icon"
  />
  <div
    className="advert-copy"
  >
    Save time grading with Premium Progress Reports
  </div>
</div>
`;
